en/editor/project/index.md

```diff
@@ -0,0 +1,86 @@
+# Project Setting
+
+The Project Settings windows are available from Cocos Creator 3D’s main menu (Project > Project Setting) which includes all the settings related to your project. These settings will be saved in the project's `settings / packages` folder. If you need to synchronize project settings between different developers, this folder should be added in your source control system.
+
+## General
+
+![general](./index/general.jpg)
+
+### Default Canvas settings
+
+The default Canvas settings include design resolution and adapted screen width/height, which are used to specify the default design resolution value in Canvas when creating a new scene or Canvas component, as well as the `Fit Height, Fit Width` options.
+
+For more information, please refer to [Multi-resolution adaptation scheme](../ui-system/components/engine/multi-resolution.md).
+
+## Engine Modules
+
+![modules](./index/modules.jpg)
+
+The setting here is to crop the modules used in the engine to reduce the size of the released engine. Modules not selected in the panel will be cropped when they are packaged and previewed.It is recommended to perform a complete test after released to avoid using cropped modules in scenes and scripts.
+
+## Macro Config
+
+For more information and code of the engine macro module, please refer to [Engine macro](https://github.com/cocos-creator/engine/blob/3d/cocos/core/platform/macro.ts#L824).
```

Comment:
For more information and code of the engine macro module, please refer to [Engine macro](https://github.com/cocos-creator/engine/blob/3d/cocos/core/platform/macro.ts#L824).
-->
For more information and code of the engine macro module, please refer to the [Engine macro](https://github.com/cocos-creator/engine/blob/3d/cocos/core/platform/macro.ts#L824) documentation.

Reply:
You missed "the"
--> please refer to the